backend/api/curious/types/index.d.ts

```diff
@@ -414,6 +414,56 @@ declare namespace curious {
     historicalCoursesPresent?: boolean
   }
 
+  /**
+   *
+   * @export
+   * @interface LearnerEducationFullDetails
+   */
+  interface LearnerEducationFullDetails {
```

Comment:
[Not to fix now - just a design suggestion]

IMO this isn't quite the right place for this definition as it isn't used by the Curious API. Given that it is created by EsweService i would define and export it there.

However i haven't looked at other type definitions here - there may be more examples of this, in which case it probably isn't worth fixing any time soon.

Reply:
I've put most of my definitions in there as I was unsure where else to put them, so it's a bigger task - though I can put something in the backlog to sort this out when the MVP is done

Reply:
ok - it isn't a disaster, just a bit misleading. Not sure it is even worth a backlog ticket, purely because we would never get round to it!